lnmuxrpc/lnmux.proto

```diff
@@ -73,3 +77,24 @@ message CancelInvoiceRequest {
 }
 
 message CancelInvoiceResponse {}
+
+message ListInvoicesRequest {
+    uint64 max_invoices_count = 1;
+    uint64 sequence_offset = 2;
```

Comment:
Nit: `sequence_start`? Offset still sounds a bit vague as to what it going to happen. But perhaps just me.